tests/test_config.py

```diff
@@ -0,0 +1,28 @@
+import os
```

Comment:
General comment, there should be some unit and integration tests for the telemetry workflow